js/client.js

```diff
@@ -26,9 +26,13 @@
 		}
 	});
 	$(window).on('dragover', function (e) {
+		if (/^text/.test(e.target.type)) // Ignore text fields
```

Comment:
Text fields are either `<input type="text">` or `<textarea>`... I'm unsure what your `/^text/.test(e.target.type)` is trying to test but it doesn't appear to be that.